Tratamento de Erros:

Atividade proposta:

O objetivo é que a função receba um array e retorne ele caso o seu tamanho corresponda ao número enviado como parâmetro na função. Caso contrário, um erro será lançado:

-Crie uma função que recebe um array e um número -Realize as seguintes validações: 1- Se os parâmetros não forem enviados, lance um erro do tipo ReferenceError 2- Se o array não for do tipo object, lance um erro do tipo TypeError 3- Se o número não for do tipo numbem, lance um erro do tipo TypeError 4- Se o tamanho do array for diferente do número enviado como parâmetro, lance um erro do tipo RangeError

-Utilize a declaração Try..Catch -Filtre as chamadas de catch por cada tipo de erro utilizando o operador instanceof.
